mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -312,6 +314,7 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 		FIELD field_23176 blockVisionPredicate Lnet/minecraft/class_4970$class_4973;
 		FIELD field_23177 postProcessPredicate Lnet/minecraft/class_4970$class_4973;
 		FIELD field_23178 emissiveLightingPredicate Lnet/minecraft/class_4970$class_4973;
+		FIELD field_25184 requiresTool Z
```

Comment:
```suggestion
		FIELD field_25184 toolRequired Z
```
This is **not** something in the builder. You argue for following the style in the builder, but this is not builder and you should use the regular naming convention.